rbac/management/migrations/0001_initial.py

```diff
@@ -16,24 +15,96 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name="Access",
             fields=[
-                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
                 ("permission", models.TextField()),
             ],
         ),
+        migrations.CreateModel(
+            name="AuditLog",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
```

Comment:
I believe for creating new table we should have new migration and we should not modify an already existing would migration 🤔